src/http.cpp

```diff
@@ -922,7 +924,17 @@ Request::handle_response_header(const asio::error_code& ec)
             std::bind(&Request::handle_response_body, this, std::placeholders::_1, std::placeholders::_2));
     }
     else if (connection_type_ == restinio::http_connection_header_t::close)
+    {
         terminate(asio::error::eof);
+    }
+    else if (response_.headers[HTTP_HEADER_TRANSFER_ENCODING] == HTTP_HEADER_TRANSFER_ENCODING_CHUNKED)
```

Comment:
`operator[]` would create `headers` item if it doesn't exist (same for line 921 https://github.com/savoirfairelinux/opendht/pull/446/files#diff-6db9a270f82169aa49daea29e8c5be13R921 ).
Prefer `find` like line 903  https://github.com/savoirfairelinux/opendht/pull/446/files#diff-6db9a270f82169aa49daea29e8c5be13R903